draft-ietf-quic-version-negotiation.md

```diff
@@ -194,92 +266,47 @@ server.
 During the handshake, endpoints will exchange handshake version information,
 which is a blob of data that is defined below. In QUIC version 1, the handshake
 version information is transmitted using a new transport parameter,
-`version_negotiation`. The contents of handshake version information depend on
-whether the client or server is sending it, and are shown below (using the
-notation from the "Notational Conventions" section of {{QUIC}}):
+`version_negotiation`. The contents of handshake version information are shown
+below (using the notation from the "Notational Conventions" section of
+{{QUIC}}):
```

Comment:
You can use `{{Section 15 of QUIC}}` if you like, or even `{{Section 15 (Versions) of QUIC}}`.  Both work (if you have a recent kramdown version).

Reply:
`{{Section 1.3 of QUIC}}` works but `{{Section 1.3 (Notational Conventions) of QUIC}}` doesn't. I'd rather have the section title when the other document isn't yet published because it could get renumbered.

FWIW it's failing with `Error: IDREF attribute target references an unknown ID "Section_1.3__Notational_Conventions__of_QUIC", at None` and I'm at https://github.com/martinthomson/i-d-template/commit/4639e5ce1002e49ee93a4bb3d9acd7644b3a83de.

Reply:
Oh I'm realizing you said a recent kramdown, not a recent i-d-template. I'll look into updating that.